scripts/testnet/include/tools.sh

```diff
@@ -5,7 +5,10 @@ startProxy() {
   setTerminalLayout "even-vertical"
 
   setWorkdirForNextCommands "$TESTNETDIR/proxy"
-  runCommandInTerminal "./proxy" $1
+  if [ "$SOVEREIGN_DEPLOY" -eq 1 ]; then
+      ARGS="--sovereign-config"
+  fi
+  runCommandInTerminal "./proxy $ARGS" $1
```

Comment:
`ARGS` is undefined unless `SOVEREIGN_DEPLOY` is true.

Reply:
+1